src/services/services.ts

```diff
@@ -375,6 +375,68 @@ namespace ts {
 
         return documentationComment;
 
+        function getCommentFromJsDocTag(tag: JSDocTag, rangeEnd: number, sourceFile: SourceFile): SymbolDisplayPart {
+            const text = sourceFile.text;
+
+            const parts: string[] = [];
+            let start = tag.end + 1;
+
+            // Eat any leading ignored characters
+            while (start < rangeEnd) {
+                const ch = text.charCodeAt(start);
+                if (ch === CharacterCodes.asterisk || isWhiteSpace(ch)) {
+                    start++;

```

Comment:
Adding the below to the `if` expression gets rid of the leading `-` in @param comments nicely for me :smile: 

```
|| (ch === CharacterCodes.minus && tag.kind === SyntaxKind.JSDocParameterTag)
```